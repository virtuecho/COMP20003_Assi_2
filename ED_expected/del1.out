COMP20003 Code,Official Code Suburb,Official Name Suburb,Year,Official Code State,Official Name State,Official Code Local Government Area,Official Name Local Government Area,Latitude,Longitude
4,50198,Buckingham,2021,5,Western Australia,51890,Collie,-33.3533990,116.3432811
104,51615,Williamstown,2021,5,Western Australia,54280,Kalgoorlie-Boulder,-30.7376701,121.4863349
577,41256,Richmond,2021,4,South Australia,48410,West Torrens,-34.9386376,138.5621269
915,11771,Greenlands,2021,1,New South Wales,17000,Singleton,-32.3614251,151.1667442
1054,41615,Windsor,2021,4,South Australia,40150,Adelaide Plains,-34.4098434,138.3147290
1117,21327,Kensington,2021,2,Victoria,"24330, 24600, 25060","Maribyrnong, Melbourne, Moonee Valley",-37.7943466,144.9270189
1625,11683,Glenroy,2021,1,New South Wales,17080,Snowy Valleys,-35.7485978,147.8827315
1986,31507,Kensington,2021,3,Queensland,31820,Bundaberg,-24.9170988,152.3182806
2158,30567,Cedar Creek,2021,3,Queensland,"33430, 34590, 36510","Gold Coast, Logan, Scenic Rim",-27.8396390,153.1844442
2273,51399,Springfield,2021,5,Western Australia,54060,Irwin,-29.2878607,114.9540971
2856,60551,Richmond,2021,6,Tasmania,61410,Clarence,-42.7387184,147.4137797
3201,22324,Springfield,2021,2,Victoria,21270,Buloke,-35.5747195,143.1143155
3525,12171,Kingswood,2021,1,New South Wales,16350,Penrith,-33.7644480,150.7249128
3670,10868,Cedar Creek,2021,1,New South Wales,17550,Tweed,-28.4183207,153.2623094
4278,12172,Kingswood,2021,1,New South Wales,17310,Tamworth Regional,-31.1608825,150.9283195
4282,13343,Red Hill,2021,1,New South Wales,17080,Snowy Valleys,-35.1584444,148.3723354
4338,33096,Windsor,2021,3,Queensland,31000,Brisbane,-27.4343963,153.0301686
4377,21966,North Melbourne,2021,2,Victoria,24600,Melbourne,-37.7984833,144.9448560
4794,51285,Red Hill,2021,5,Western Australia,58050,Swan,-31.8275678,116.0745512
4898,40678,Kensington Gardens,2021,4,South Australia,40700,Burnside,-34.9197249,138.6631866
5120,10125,Back Creek,2021,1,New South Wales,13660,Gwydir,-30.1122498,150.2755951
5147,50031,Ascot,2021,5,Western Australia,50490,Belmont,-31.9340902,115.9294786
5262,22793,Williamstown North,2021,2,Victoria,23110,Hobsons Bay,-37.8526203,144.8689429
5446,22256,Seddon,2021,2,Victoria,24330,Maribyrnong,-37.8066284,144.8917067
5496,32426,Richmond Hill,2021,3,Queensland,32310,Charters Towers,-20.0640332,146.2649959
5549,10867,Cedar Creek,2021,1,New South Wales,11720,Cessnock,-32.8372268,151.1719041
5841,13012,North Richmond,2021,1,New South Wales,13800,Hawkesbury,-33.5680371,150.7204085
5926,13344,Red Hill,2021,1,New South Wales,17310,Tamworth Regional,-30.4937039,150.6341084
6139,11772,Greenlands,2021,1,New South Wales,"10550, 17040","Bega Valley, Snowy Monaro Regional",-36.5179479,149.4438115
6165,11682,Glenroy,2021,1,New South Wales,10050,Albury,-36.0515044,146.8993683
6206,50752,Kensington,2021,5,Western Australia,"57840, 58510","South Perth, Victoria Park",-31.9856189,115.8839737
6261,21641,Melbourne Airport,2021,2,Victoria,23270,Hume,-37.6737537,144.8380142
6363,32629,Springfield Central,2021,3,Queensland,33960,Ipswich,-27.6838705,152.9057688
6466,30568,Cedar Creek,2021,3,Queensland,35010,Moreton Bay,-27.3337142,152.8183012
6499,10128,Back Creek,2021,1,New South Wales,17400,Tenterfield,-28.9617361,151.7290031
6623,21640,Melbourne,2021,2,Victoria,"24600, 25900, 26350, 27350","Melbourne, Port Phillip, Stonnington, Yarra",-37.8249613,144.9715278
6636,13640,Springfield,2021,1,New South Wales,17040,Snowy Monaro Regional,-36.5324737,149.1239824
7035,32401,Red Hill,2021,3,Queensland,37310,Western Downs,-26.6439922,150.6450088
7057,14343,Windsor,2021,1,New South Wales,13800,Hawkesbury,-33.6090236,150.8211213
7302,40510,Glenroy,2021,4,South Australia,48340,Wattle Range,-37.2100599,140.7793209
7333,22757,West Melbourne,2021,2,Victoria,24600,Melbourne,-37.8093501,144.9251752
7502,60100,Carlton River,2021,6,Tasmania,64810,Sorell,-42.8642702,147.7112742
7720,12107,Kensington,2021,1,New South Wales,16550,Randwick,-33.9134157,151.2207450
7750,12170,Kingswood,2021,1,New South Wales,10550,Bega Valley,-36.7422084,149.8221413
7903,80112,Red Hill,2021,8,Australian Capital Territory,89399,Unincorporated ACT,-35.3334057,149.1206203
8133,13617,South Windsor,2021,1,New South Wales,13800,Hawkesbury,-33.6300880,150.8049816
8138,21809,Mount Richmond,2021,2,Victoria,22410,Glenelg,-38.1544120,141.3324009
8161,41325,Seddon,2021,4,South Australia,42750,Kangaroo Island,-35.8589260,137.2800867
8205,30089,Ascot,2021,3,Queensland,31000,Brisbane,-27.4298286,153.0652855
8540,32400,Red Hill,2021,3,Queensland,31000,Brisbane,-27.4525092,153.0025949
8552,32424,Richmond,2021,3,Queensland,34770,Mackay,-21.0869211,149.1405356
8972,22805,Windsor,2021,2,Victoria,"24600, 25900, 26350","Melbourne, Port Phillip, Stonnington",-37.8546754,144.9932064
9024,40044,Ascot Park,2021,4,South Australia,44060,Marion,-34.9881165,138.5610086
9032,10126,Back Creek,2021,1,New South Wales,15240,Mid-Coast,-31.9616852,152.0531687
9130,22148,Red Hill,2021,2,Victoria,25340,Mornington Peninsula,-38.3635428,145.0160941
9272,20073,Ascot,2021,2,Victoria,20570,Ballarat,-37.3964624,143.8130771
9414,41602,Williamstown,2021,4,South Australia,40310,Barossa,-34.6689498,138.8960892
9602,31173,Glenroy,2021,3,Queensland,36370,Rockhampton,-23.1851451,149.8175691
9726,31508,Kensington Grove,2021,3,Queensland,34580,Lockyer Valley,-27.5327853,152.4721060
9773,20495,Carlton,2021,2,Victoria,24600,Melbourne,-37.8004392,144.9680900
9900,12357,Little Back Creek,2021,1,New South Wales,14550,Kyogle,-28.6134201,153.0458267
9993,10129,Back Creek,2021,1,New South Wales,17550,Tweed,-28.4090983,153.1702791
10331,22038,Parkville,2021,2,Victoria,"24600, 25060, 25250","Melbourne, Moonee Valley, Moreland",-37.7866675,144.9512266
10486,40677,Kensington,2021,4,South Australia,45290,Norwood Payneham and St Peters,-34.9237458,138.6456183
10671,40702,Kingswood,2021,4,South Australia,44340,Mitcham,-34.9708654,138.6137673
10939,20830,East Melbourne,2021,2,Victoria,24600,Melbourne,-37.8143552,144.9826486
11249,10124,Back Creek,2021,1,New South Wales,10800,Bland,-33.8645950,147.4605038
11480,30405,Buckingham,2021,3,Queensland,30900,Boulia,-22.2071693,139.5282922
11537,40679,Kensington Park,2021,4,South Australia,40700,Burnside,-34.9201888,138.6542865
11627,10818,Carlton,2021,1,New South Wales,"10500, 12930","Bayside, Georges River",-33.9710214,151.1213916
12364,22792,Williamstown,2021,2,Victoria,23110,Hobsons Bay,-37.8611868,144.8899758
12417,13375,Richmond,2021,1,New South Wales,13800,Hawkesbury,-33.6048433,150.7574251
12526,22310,South Melbourne,2021,2,Victoria,25900,Port Phillip,-37.8339379,144.9572894
12570,30090,Ascot,2021,3,Queensland,36910,Toowoomba,-27.8254088,151.9968069
12786,13345,Red Hill,2021,1,New South Wales,17950,Warren,-31.7345343,147.7943696
12852,32425,Richmond,2021,3,Queensland,36300,Richmond,-20.7622455,143.2365906
12999,32630,Springfield Lakes,2021,3,Queensland,33960,Ipswich,-27.6854087,152.9178969
13321,14344,Windsor Downs,2021,1,New South Wales,13800,Hawkesbury,-33.6559424,150.8082438
13552,13378,Richmond Vale,2021,1,New South Wales,11720,Cessnock,-32.8651705,151.4971983
13842,60540,Red Hills,2021,6,Tasmania,64210,Meander Valley,-41.5290479,146.5825223
14412,20074,Ascot,2021,2,Victoria,22620,Greater Bendigo,-36.7091600,144.3352295
14446,21047,Glenroy,2021,2,Victoria,25250,Moreland,-37.7026618,144.9266889
14507,31238,Greenlands,2021,3,Queensland,36660,Southern Downs,-28.6510732,151.7887354
14539,60099,Carlton,2021,6,Tasmania,64810,Sorell,-42.8609790,147.6536575
14551,10127,Back Creek,2021,1,New South Wales,16490,Queanbeyan-Palerang Regional,-35.3433327,149.9435424
14736,40177,Buckingham,2021,4,South Australia,47630,Tatiara,-36.3381207,140.5673969
14821,41571,West Richmond,2021,4,South Australia,48410,West Torrens,-34.9393884,138.5503648
14926,22170,Richmond,2021,2,Victoria,27350,Yarra,-37.8202671,145.0024290
15157,13376,Richmond Hill,2021,1,New South Wales,14850,Lismore,-28.7892790,153.3469865
15169,41616,Windsor Gardens,2021,4,South Australia,45890,Port Adelaide Enfield,-34.8654824,138.6543285
15232,13165,Parkville,2021,1,New South Wales,17620,Upper Hunter Shire,-31.9705195,150.8823423
