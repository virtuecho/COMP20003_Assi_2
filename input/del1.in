Spotswood
Richmond Plains
New York
Ascot Vale
Docklands
South
Richmond Lowlands
Port Melbourne
Far Far Away
Yarraville
Red Hill South
Carlton North
