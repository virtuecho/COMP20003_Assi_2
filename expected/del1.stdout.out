query= Spotswood
Spotswood --> 1 records deleted
query= Richmond Plains
Richmond Plains --> 1 records deleted
query= New York
New York --> NOT FOUND
query= Ascot Vale
Ascot Vale --> 1 records deleted
query= Docklands
Docklands --> 1 records deleted
query= South
South --> NOT FOUND
query= Richmond Lowlands
Richmond Lowlands --> 1 records deleted
query= Port Melbourne
Port Melbourne --> 1 records deleted
query= Far Far Away
Far Far Away --> NOT FOUND
query= Yarraville
Yarraville --> 1 records deleted
query= Red Hill South
Red Hill South --> 1 records deleted
query= Carlton North
Carlton North --> 1 records deleted
Summary: 12 deleted operations performed, 9 records deleted in total
